src/expand.rs

```diff
@@ -357,8 +357,8 @@ impl ExpandedTest {
                 return Ok(ExpansionOutcome::NoExpandedFileFound);
             }
 
-            // Write a .expanded.rs file contents with an newline character at the end
-            std::fs::write(expanded, &format!("{}\n", output))?;
```

Comment:
This change was necessary to get the existing tests passing. Probably broken by #78 (sorry). `prettyplease` already ends every line (including last line) with a newline.